src/math/vec.zig

```diff
@@ -38,6 +41,9 @@ pub fn Vec(comptime n_value: usize, comptime Scalar: type) type {
                 pub inline fn init(xs: Scalar, ys: Scalar, zs: Scalar) VecN {
                     return .{ .v = .{ xs, ys, zs } };
                 }
+                pub inline fn fromInt(xs: i128, ys: i128, zs: i128) VecN {
```

Comment:
@omprakaash by using `i128` here I think we may be forcing the compiler to use more memory, or at the very least relying on the optimizer to notice that we aren't doing that.

A better approach here might be to use `anytype` instead of `i128`, then anything that is an integer can be passed in and `@floatFromInt` will act however it normally acts when an arbitrary type is given to it.

Reply:
Ah got you. Was not thinking about `anytype`. Will change the signature to include it.